net/instaweb/rewriter/css_inline_filter.cc

```diff
@@ -97,7 +97,8 @@ class CssInlineFilter::Context : public InlineRewriteContext {
 CssInlineFilter::CssInlineFilter(RewriteDriver* driver)
     : CommonFilter(driver),
       id_(RewriteOptions::kCssInlineId),
-      size_threshold_bytes_(driver->options()->css_inline_max_bytes()) {
+      size_threshold_bytes_(driver->options()->css_inline_max_bytes()),
+      inlining_not_pedantically_spec_(false) {
```

Comment:
This is actually a document-state bit, which I didn't realize until I saw where it was spec.  So you need to init it in StartDocumentImpl, as the same CssInlineFilter class instance can be re-used for multiple documents (we recycle drivers for compatible configurations).

Also I think I would call the bit has_pedantic_inline_violations_.